index.html

```diff
@@ -87,112 +133,114 @@ <h3>LET'S CONNECT</h3>
-        />
+          src="Snapshoot_Portfolio.png"
+       />
         <div class="project-section">
-          <h2 class="project-name-two">Tonic</h2>
-          <div class="project-info-two">
+          <h2 class="project-name">Tonic</h2>
+          <nav class="project-info">
             <h4 class="project-client">CANOPY</h4>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="programming-language">Back End Dev</span>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="year">2015</span>
-          </div>
-          <p class="project-details-two">
-            A daily selection of privately personalized reads; no accounts or<br />
+          </nav>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
             sign-ups required.
           </p>
-          <ul class="languages-two">
+          <ul class="languages">
             <li class="dev-tool">html</li>
             <li class="dev-tool">css</li>
             <li class="dev-tool">javascript</li>
           </ul>
-          <button class="btn" type="button">See Project</button>
+          <button type="button" class="btn">See Project</button>
         </div>
       </div>
 
       <div class="portfolio-border">
         <img
           class="portfolio-image"
           alt="portfolio project"
-          src="Portfolio2.png"
+          src="Portfolio4.png"
         />
-        <div class="project-section">
-          <h2 class="project-name-three">Multi-Post <br /> Stories </h2>
-          <div class="project-info-three">
+        <div class="project-section project-section-reverse">
+          <h2 class="project-name">Multi-Post Stories</h2>
+          <nav class="project-info">
             <h4 class="project-client">CANOPY</h4>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="programming-language">Back End Dev</span>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="year">2015</span>
-          </div>
-          <p class="project-details-three">
-            A daily selection of privately personalized reads; no accounts or<br />
+          </nav>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
             sign-ups required.
           </p>
-          <ul class="languages-zero">
+          <ul class="languages">
             <li class="dev-tool">html</li>
             <li class="dev-tool">css</li>
             <li class="dev-tool">javascript</li>
           </ul>
-          <button class="btn" type="button">See Project</button>
+          <button type="button" class="btn">See Project</button>
         </div>
       </div>
-    </section>    
-    <section class="about-me-section" id="about">
+    </section>
+
+    
+    <section class="about-me" id="about">
```

Comment:
Kindly note that the left part of the "About myself" section takes 40% of the screen width and the right portion takes 60% width as shown below. So please fix that 🤗
Also kindly fix the margin on the right side as mentioned in the image below 

Your design:
![image](https://user-images.githubusercontent.com/73714615/186905663-4dc5026b-64d8-419f-894c-0b4558518e91.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186905722-f2cf1e07-a0e7-4813-9840-94529a117517.png)